test/test_mahalanobis_mixin.py

```diff
@@ -194,7 +194,7 @@ def test_get_metric_equivalent_to_explicit_mahalanobis(estimator,
   a, b = (rng.randn(n_features), rng.randn(n_features))
   expected_dist = mahalanobis(a[None], b[None],
                               VI=model.get_mahalanobis_matrix())
-  assert_allclose(metric(a, b), expected_dist, rtol=1e-15)
+  assert_allclose(metric(a, b), expected_dist, rtol=1e-13)
```

Comment:
This is to fix the test that didn't pass in this build: https://travis-ci.org/metric-learn/metric-learn/builds/531766687?utm_source=github_status&utm_medium=notification 
I didn't manage to reproduce the bug locally either (it passes for me, with the same versions of packages than travis, so I don't know what may be going wrong...)
But I guess 1e-13 is still a reasonable tolerance